src/main/java/uk/gov/ons/fwmt/job_service/service/impl/JobProcessor.java

```diff
@@ -162,4 +107,71 @@ public void processSampleFile(File file) throws IOException{
       return Optional.empty();
     }
   }
+
+  protected void sendJobToUser(int row, LegacySampleIngest ingest, UserDto userDto, boolean isReallocation) {
+    if (jobResourceServiceClient.existsByTmJobIdAndLastAuthNo(ingest.getTmJobId(), userDto.getAuthNo())) {
+      log.error("Job Entry could not be processed for row: {}", row, FWMTCommonException.makeCsvOtherException("Job has been sent previously"));
+      return;
+    }
+
+    if (isReallocation) {
+      processReallocation(ingest, userDto);
+    } else {
+      processBySurveyType(ingest, userDto, row);
+    }
+  }
+
+  protected void processReallocation(LegacySampleIngest ingest, UserDto userDto) {
+    final SendUpdateJobHeaderRequestMessage request = tmJobConverterService.updateJob(ingest, userDto.getTmUsername());
+    log.info("Reallocating job with ID {} to user {}", ingest.getTmJobId(), userDto.toString());
+    tmService.send(request);
+    final Optional<JobDto> jobDto = jobResourceServiceClient.findByTmJobId(ingest.getTmJobId());
+
+    jobDto.ifPresent(jobDto1 -> {
+      jobDto1.setLastAuthNo(ingest.getAuth());
+      jobResourceServiceClient.updateJob(jobDto1);
+    });
+  }
+
+  protected void processBySurveyType(LegacySampleIngest ingest, UserDto userDto, int row) {
+    LocalDateTime lastUpdateParsed = null;
+
+    try {
+      lastUpdateParsed = LocalDateTime.parse(ingest.getLastUpdated(), DateTimeFormatter.ISO_LOCAL_DATE_TIME);
+    } catch (Exception e) {
+      log.error("Job Entry could not be processed for row: {}", row, FWMTCommonException.makeCsvOtherException("Last updated column cannot be parsed")); //Wierd shouldnt Exceptions be thrown
+      return;
+    }
+
+    switch (ingest.getLegacySampleSurveyType()) {
+    case GFF:
+      processGFFSample(ingest, userDto, lastUpdateParsed);
+      break;
+    case LFS:
+      processLFSSample(ingest, userDto, lastUpdateParsed);
+      break;
+    default:
+      throw new IllegalArgumentException("Unknown survey type");
+    }
+  }
+
+  protected void processLFSSample(LegacySampleIngest ingest, UserDto userDto, LocalDateTime lastUpdateParsed) {
+    final SendCreateJobRequestMessage request = tmJobConverterService.createJob(ingest, userDto.getTmUsername());
+    log.info("Reissuing GFF job with ID {} to user {}", ingest.getTmJobId(), userDto.toString());
```

Comment:
log.info needs changing to create LFS job.
There is no reissue in LFS. missed out in initial review probably.